core-processor/src/executor.rs

```diff
@@ -141,18 +140,18 @@ where
     let execute = || {
         let env = Environment::new(
             ext,
-            program.code_bytes(),
+            program.code.code(),
```

Comment:
to consider renaming here since having `fn code` for `struct Code` looks weird

Reply:
This would change too many files, so I will make issue to make this in other PR

Reply:
Because we have the same method in `core::code::Code`

Reply:
https://github.com/gear-tech/gear/issues/3985